linetools/spectra/tests/test_xspectrum_utils.py

```diff
@@ -149,6 +149,8 @@ def test_copy(spec):
     assert spec.wavelength[0] == spec2.wavelength[0]
     assert spec.flux[-1] == spec2.flux[-1]
 
+def test_plot(spec):
```

Comment:
How about we separate all the tests which require xvfb into their own file? Maybe test_plotting.py?